### DISCUSSION TITLE
Checkbox To Manually Trigger Renovate Removed From Dependency Dashboard

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#20386] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

We are using Forking Renovate with the Dependency Dashboard enabled on [all of our public repositories](https://github.com/ScribeMD/). There used to be a checkbox at the bottom of the Dependency Dashboard issue labelled "Check this box to trigger a request for Renovate to run again on this repository." It disappeared from all of our repositories yesterday. This was a very valuable feature, particularly now that the typical frequency of automatic Renovate runs appears to have skyrocketed from 3-4 hours to at least 12. Was this feature intentionally removed? If so, is there any alternate way to trigger a Renovate run without opening a PR? The drawback to opening a random PR is that it results in unwanted noise, particularly when a specific upgrade that doesn't yet appear in the list needs to happen first in order to fix the build.

### What you tried so far.

I believe opening a random PR effectively manually triggers Renovate, updating the list of available upgrades. https://developer.mend.io/github/ScribeMD gives a 500, internal server error, so I am unable to provide logs at the moment.

**Comment by nabeelsaabna** [comment, marked as answer]
@Kurt-von-Laven thanks for raising the issue, we are looking into it.

**Reply by Gabriel-Ladzaretti** [reply]
Should be fixed now, tested on my end.
next time Renovate runs the checkbox should be back. 
Thanks again for reporting this 😄

**Reply by Kurt-von-Laven** [reply]
Thank you! The checkbox is already back on all of our repositories, so it appears that Renovate is running more frequently again as well!
